dbms/src/TestUtils/TiFlashTestEnv.h

```diff
@@ -69,8 +69,8 @@ class TiFlashTestEnv
 
     static Context getContext(const DB::Settings & settings = DB::Settings(), Strings testdata_path = {});
 
-    static void initializeGlobalContext(Strings testdata_path = {}, PageStorageRunMode ps_run_mode = PageStorageRunMode::ONLY_V3, uint64_t bg_thread_count = 0);
-    static void addGlobalContext(Strings testdata_path = {}, PageStorageRunMode ps_run_mode = PageStorageRunMode::ONLY_V3, uint64_t bg_thread_count = 0);
+    static void initializeGlobalContext(Strings testdata_path = {}, PageStorageRunMode ps_run_mode = PageStorageRunMode::ONLY_V3, uint64_t bg_thread_count = 2);
+    static void addGlobalContext(Strings testdata_path = {}, PageStorageRunMode ps_run_mode = PageStorageRunMode::ONLY_V3, uint64_t bg_thread_count = 2);
```

Comment:
why change these two default value?

Reply:
By default we spawn too many background threads per test case. When executed concurrently (e.g. on CI we use parallel = 12, which means we will open 12 × 32 threads) it is a waste of system resource.

Reply:
Get it